_posts/2020-10-10-temperature-water-content-and-polishing-effects-on-koji.md

```diff
@@ -43,14 +43,16 @@ As an aside, this was the experimental set-up:
 ![Figure 1](/assets/images/temp_effect/Figure 1.png){:.responsive .center}
 *The experimental setup - see Ref. 2*
 
-*Callout:* This setup is a sort of respirometer - I'm not sure of the exact name. Oxygen is produced by electrolysis, which builds up pressure in the system, pushing one side of the mercury manometer down. With enough pressure, the mercury level switch opens, stopping the generation of oxygen. The koji continues consuming oxygen, and one can measure total oxygen consumption by integrating the total current consumed over time. Finally, waste carbon dioxide is absorbed by the NaOH solution.
+<div class="callout">
+💡 This setup is a sort of respirometer - I'm not sure of the exact name. Oxygen is produced by electrolysis, which builds up pressure in the system, pushing one side of the mercury manometer down. With enough pressure, the mercury level switch opens, stopping the generation of oxygen. The koji continues consuming oxygen, and one can measure total oxygen consumption by integrating the total current consumed over time. Finally, waste carbon dioxide is absorbed by the NaOH solution.
+</div>
 
```

Comment:
Using a callout class to separate out the experimental setup

Reply:
cool!